### ISSUE TITLE
More style names could be detected as code by the ODT reader

### ISSUE THREAD

**Issue by danse** [jgm/pandoc#4841]
In order to detect code formatting in an ODT document, pandoc currently recognises the `Source_Text` value for the `style-name` attribute, but this is not the value used by the styles i have available with Libreoffice 5. I can use _Preformatted Text_ for paragraphs and _Source Text_ among the "character styles" that is inlines. They get encoded as `Preformatted_20_Text` and `Source_20_Text` values for the style names.

**Comment by jgm** [comment]
I suspect this would be an easy change; just search
for "Source_Text" in the code...

**Comment by danse** [comment]
yes it's easy to add more styles. I'm also looking for a way to convert code paragraphs, currently it doesn't works. Hopefully i will close this issue fixing both problems
